### PR TITLE
feat(core): add bullet graph

### PR DESCRIPTION
**Pull request by theiliad**
image

### CODE REVIEW

**Comment by DianaStanciulescu** [comment]
1. i think we should limit to 3 unless we get push against it (we run into some accessibility issues if we don;t)

2. there is an option without any area - that space is shown like a simple rectange
Screenshot 2021-03-12 at 21 46 57

3. no, they can be called whatever people need to call them.

**Review by sophiiae — DISMISSED** [review]

File: packages/core/demo/data/bullet.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { mouse } from 'd3';
```
Comment (natashadecoste) [inline packages/core/demo/data/bullet.ts:1]:
```suggestion
```

**Review by natashadecoste — CHANGES_REQUESTED** [review]
the tooltip _values_ are correct, but the color flag seems to only ever use the first default purple
image

**Comment by theiliad** [comment]
> the tooltip _values_ are correct, but the color flag seems to only ever use the first default purple

fixed

**Comment by DianaStanciulescu** [comment]
hi @theiliad , question: why are the X axis values slanted, and if the need to be, why is 0 slanted the other way?

**Comment by DianaStanciulescu** [comment]
also, the quartile marks on top of the colored bar should be black in the dark theme, and white when they are outside the colored bars.

**Review by cal-smith — DISMISSED** [review]

**Comment by cal-smith** [comment]
Just a code review on my part 👍

**Comment by theiliad** [comment]
> hi @theiliad , question: why are the X axis values slanted, and if the need to be, why is 0 slanted the other way?

the slanting is a bug in the axis charts where in some cases the labels (that do fit) are still slanted, have to fix that

**Review by sophiiae — APPROVED** [review]

**Comment by theiliad** [comment]
Have to convert to draft again since we're missing:
1. correct legend behaviour #725 
2. wrappers

**Comment by mjabbink** [comment]
@theiliad I think the inside chart background should be white when on G10 background. 

Screen Shot 2021-03-22 at 3 52 30 PM

**Comment by mjabbink** [comment]
Currently all G10.

Screen Shot 2021-03-22 at 3 52 54 PM

**Comment by mjabbink** [comment]
Same for dark G90 dark theme

Screen Shot 2021-03-22 at 3 57 37 PM


Screen Shot 2021-03-22 at 3 57 52 PM

**Comment by mjabbink** [comment]
@DianaStanciulescu  :point_up:

**Comment by jeanservaas** [comment]
Agree about live chart background. I'm pretty sure this was just an oversight and not intentional as far as I can remember. 

So I'm assuming that there is still tone in those first sections, like this:
image

**Comment by mjabbink** [comment]
I wonder in the case of this chart it’s better as it is? Seems almost to get more busy following the suggestion?

**Comment by DianaStanciulescu** [comment]
yes, the bg was not intentional, it should be indeed white. i worked on completely white BG so I missed pointing this out. 

i guess they all look busier with white bg vs g10 general bg,  but at the same time maybe more legible, better contrast, etc?

**Comment by mjabbink** [comment]
And probably more typically on white or G100 but totally guessing. Also best to stay consistent. We can change if we think it will hinder readability later on down the line.

**Review by sophiiae — DISMISSED** [review]

**Review by sophiiae — APPROVED** [review]

**Comment by theiliad** [comment]
> @DianaStanciulescu ☝️ FYI @jeanservaas

I just added the BGs on g10 & g90

**Review by natashadecoste — DISMISSED** [review]

**Review by sophiiae — APPROVED** [review]

**Review by natashadecoste — APPROVED** [review]

**Pull request merged**